packages/website/.scripts/docusaurus-build.mjs

```diff
@@ -1,11 +1,24 @@
 #!/usr/bin/env node
 // @ts-check
 
+import dotenv from "dotenv";
+import path from "path";
+import { fileURLToPath } from "url";
 import { run } from "../../../eng/scripts/helpers.js";
 
+loadDotenv();
+
 if (process.env.TYPESPEC_SKIP_DOCUSAURUS_BUILD?.toLowerCase() === "true") {
   console.log("Skipping docusaurus build: TYPESPEC_SKIP_DOCUSAURUS_BUILD=true");
   process.exit(0);
 }
 
 run("docusaurus", ["build"]);
+
+function loadDotenv() {
+  const dirname = path.dirname(fileURLToPath(import.meta.url));
+  const dotenvPath = path.resolve(dirname, "../../../.env");
```

Comment:
should it be able to figure out if it is inside typespec-azure repo by checking one folder above too

Reply:
Added logic